src/test/java/BaseTest.java

```diff
@@ -1,11 +1,89 @@
+    By playNextSongButton = By.cssSelector("i.next.fa.fa-step-forward.control");
+    //By playSongButton = By.xpath("//*[@title='Play or resume' and  @data-testid='play-btn']");
+    By playSongButton = By.cssSelector("span.play i.fa.fa-play");
+
+    //By pauseButton = By.xpath("//span[@title ='Pause' and @data-testid='pause-btn']");
+    By pauseButton = By.cssSelector("span.pause i.fa.fa-pause");
 
+    //By soundBarVisualizer = By.cssSelector("[data-testid='sound-bar-play']");
+    By soundBarVisualizer = By.cssSelector("div.bars");
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
+    @BeforeMethod
+    public void setUpBrowser() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--disable-notifications","--remote-allow-origins=*");
+        driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(20));
+    }
+    @AfterMethod
+    public void tearDownBrowser() {
+        driver.quit();
+    }
+//    public void openLoginURL() {
+//        driver.get(url);
+//    }
+//    public void provideEmail(String Email) {
+//        WebElement emailFieldElement = driver.findElement(emailField);
+//        emailFieldElement.sendKeys(Email);
+//    }
+//    public void providePW(String Password) {
+//        WebElement passwordFieldElement = driver.findElement(passwordField);
+//        passwordFieldElement.sendKeys(Password);
+//    }
+//    public void clickSubmit() {
+//        WebElement submitButtonElement = driver.findElement(submitButton);
+//        submitButtonElement.click();
+//    }
+    public void login(String Email, String Password){
+        driver.get(url);
+        WebElement emailFieldElement = driver.findElement(emailField);
+        emailFieldElement.sendKeys(Email);
+        WebElement passwordFieldElement = driver.findElement(passwordField);
+        passwordFieldElement.sendKeys(Password);
+        WebElement submitButtonElement = driver.findElement(submitButton);
+        submitButtonElement.click();
+    }
+    public void clickPlayNextSong() throws InterruptedException{
+        WebElement playNextSongElement = driver.findElement(playNextSongButton);
+        playNextSongElement.click();
+        System.out.println("Play Next Song Clicked");
+        Thread.sleep(2000);
+    }
+   public void clickPlaySong() throws InterruptedException{
+        WebElement PlaySongElement = driver.findElement(playSongButton);
+        PlaySongElement.click();
+        System.out.println("Play Song Clicked");
+       Thread.sleep(2000);
+    }
+    public boolean pauseButtonDisplay() throws InterruptedException {
+        WebElement pauseButtonElement = driver.findElement(pauseButton);
+        Thread.sleep(2000);
+        return pauseButtonElement.isDisplayed();
+    }
+    public boolean soundBarVisualizerDisplay() throws InterruptedException{
```

Comment:
By the way, I recommend you to design the code according to the Page Object Model. That is, each component/page encapsulates all the actions and properties of that component/page.

// HINT: divided into separate classes like `LoginPage`, `HomePage`, ect.. 
should not be grouped together in the same `BaseTest` class.
-> This will make it easier to maintain, reuse, and read the code